app/assets/javascripts/services/miq_service.js

```diff
@@ -124,11 +124,14 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', '$log
 
     return serializedObj;
   };
+
   this.handleFailure = function(e) {
     miqSparkleOff();
+
     if (e.message) {
-      $log.log(e.message);
+      console.error(e.message);
     }
```

Comment:
I don't see a reason to remove `$log` here.

How about using `$log.error()` instead of `console.error` (will also take care of the CC error)